packages/now-node/src/helpers.ts

```diff
@@ -72,6 +73,46 @@ function getCookieParser(req: NowRequest) {
   };
 }
 
+function parseQSArray(key: string, value: any, acc: any): void {
+  const bktNotation = /\[[^[\]\d]?]/.test(key);
```

Comment:
What does `bktNotation` represent? Perhaps it could be `isBrackets`?

Reply:
This represents whether the array is formed using bracket notation (`item[]=value1&item[]=value2`) or index notation (`item[0]=value1&item[1]=value2`)

Reply:
Ok I see now.

I think there are three possibilities that need to be accounted for:

1. No key`item[]=value1&item[]=value2` becomes an array
2. Number key`item[0]=value1&item[1]=value2` becomes an array
3. String key `item[key1]=value1&item[key2]=value2` becomes an object